src/components/SearchFilters.vue

```diff
@@ -88,61 +39,15 @@
 </template>
 
 <script>
+import { computed } from '@vue/composition-api';
+import SelectFilter from '@/components/helpers/SelectFilter';
```

Comment:
This seems an actual component, not a helper. Perhaps we should not have this in a helper folder, as thus far helpers are mere functions. What do you reckon'?

Reply:
It is not the first component to end up in the helper folder. Feel free to improve the folder structure.

Reply:
Good enough for now.